crates/oxc_prettier/src/format/mod.rs

```diff
@@ -1341,7 +1342,7 @@ impl<'a> Format<'a> for NumericLiteral<'a> {
             let mut string = Cow::Borrowed(raw);
 
             if string.contains(|c: char| c.is_ascii_uppercase()) {
```

Comment:
Why was this `is_ascii_uppercase` but the code is `cow_to_ascii_lowercase` 🤣 

But these `contains` check can be removed.

Reply:
remove all or remove this?